### PR TITLE
Npg 3283 explorer stake pool test

### PR DESCRIPTION
**Pull request by kukkok3**

### CODE REVIEW

File: src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+            stake_pool.info().operators.len(),
+            explorer_stake_pool.registration.operators.len()
+        );
+
+        assert_eq!(
+            stake_pool.info().operators.len(),
+            explorer_stake_pool.registration.operators.len()
+        );
+        let mut operators_matching = 0;
+        for operator in &explorer_stake_pool.registration.operators {
```
Comment (dkijania) [inline src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs:34]:
```
assert(explorer_stake_pool.registration.operators.iter().all(|operator| stake_pool
                .info()
                .operators
                .contains(&Self::decode_bech32_pk(operator))));
```

File: src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+                        stake_pool.info().operators.len(),
+                        explorer_stake_pool.registration.operators.len()
+                    );
+
+                    assert_eq!(
+                        stake_pool.info().operators.len(),
+                        explorer_stake_pool.registration.operators.len()
+                    );
+                    let mut operators_matching = 0;
+                    for operator in &explorer_stake_pool.registration.operators {
```
Comment (dkijania) [inline src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs:174]: the same as above

File: src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+                        }
+                    }
+                    assert_eq!(stake_pool.info().operators.len(), operators_matching);
+
+                    assert_eq!(
+                        stake_pool.info().owners.len(),
+                        explorer_stake_pool.registration.owners.len()
+                    );
+                    let mut owners_matching = 0;
+                    for owner in &explorer_stake_pool.registration.owners {
```
Comment (dkijania) [inline src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs:190]: here too

File: src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs (hunk 1 above)
Comment (kukkok3) [reply src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs:34]: Nice one, that's what I was looking for!

File: src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+                        explorer_stake_pool
+                            .registration
+                            .rewards
+                            .ratio
+                            .numerator
+                            .parse::<u64>()
+                            .unwrap()
+                    );
+                    assert_eq!(
+                        stake_pool.info().rewards.ratio.denominator,
```
[the remark this answers is not in the document]
Comment (kukkok3) [reply src/jormungandr/testing/jormungandr-automation/src/jormungandr/explorer/verifiers/stake_pool_verifier.rs:236]: No, one assert is for numerator the other one is for denominator

**Review by dkijania — APPROVED** [review]

**Pull request merged**